src/app/docs/list-filters/demo/list-filters-demo-modal.component.html

```diff
@@ -0,0 +1,68 @@
+<sky-modal>
+  <sky-modal-header>
+    {{headerText}}
+  </sky-modal-header>
+  <sky-modal-content>
+    <label
+      [for]="fruitTypeSelect.id"
+    >
```

Comment:
```suggestion
      class="sky-control-label"
    >
```
The label seems too tight to the input.